docs/resources/useful-resources.md

```diff
@@ -2,11 +2,38 @@
 
 🔔 Looking for [accessibility docs?](../accessibility-docs)
 
-## Standards and resources
+One of our goals is to collect information about accessibility and create documentation to empower the Jupyter community to help us make our software accessible. Please use this section to begin to aggregate relevant accessibility third party standards, guides, and documentation.
 
-One of our goals is to collect information about accessibility and create documentation to empower the Jupyter community
-to help us make our software accessible. Please use this section to begin to aggregate relevant third party standards,
-guides, and documentation related to this:
+## Accessibility guidelines
+
+The World Wide Web Consortium (W3C) publishes a number of international recognized technical guidelines, like the Web Accessibility Initiative (WAI) and XHTML standards. These guidelines assure that a given website can be accessed and consumed by as many people as possible.
+
+### Accessible Rich Internet Applications (WAI-ARIA)
+
+[WAI-ARIA](https://www.w3.org/WAI/standards-guidelines/aria/) provides a route to make Web content and applications more accessible, it specifically targets dynamic content and advanced user interface controls that are normally developed in JavaScript, HTML and other related technologies. A website that doesn't implement this guideline will not be accessible to some users with disabilities, specially the ones that rely on a screen reader or cannot use the mouse.
```

Comment:
Sometimes `web` is written capitalized as `Web` even when it is not part of a proper noun. I'm not sure if this is intentional and I'm misunderstanding the sentence, or this needs to be lowercased. I am seeing this only in Line 13 and 15.

With how many Proper Web Nouns are accurately in this document, I'm impressed at how many of them seem totally right!

Reply:
I think we can lowercase in both lines